api/v1alpha1/upgradeplan_types.go

```diff
@@ -28,6 +28,7 @@ const (
 	LonghornUpgradedCondition   = "LonghornUpgraded"
 	MetalLBUpgradedCondition    = "MetalLBUpgraded"
 	CDIUpgradedCondition        = "CDIUpgraded"
+	KubevirtUpgradedCondition   = "KubevirtUpgraded"
```

Comment:
```suggestion
	KubevirtUpgradedCondition   = "KubeVirtUpgraded"
```

Nit: Can we use the official spelling here?